extensions/stackdriver/stackdriver.cc

```diff
@@ -403,6 +404,12 @@ bool StackdriverRootContext::recordTCP(uint32_t id) {
   // Record TCP Metrics.
   ::Extensions::Stackdriver::Metric::recordTCP(outbound, local_node, peer_node,
                                                request_info);
+  // Add LogEntry to Logger. Log Entries are batched and sent on timer
+  // to Stackdriver Logging Service.
+  if (enableTCPServerAccessLog() && shouldLogThisRequest()) {
```

Comment:
Does `shouldLogThisRequest` apply here? It only bases on http or grpc status right?

Reply:
Removed for now.. but I was thinking more in terms of sampling successful connections too but will change accesslog sampling filter to select failed request based on presence of response flag.. will do in a separate PR